roles/analytics-worker/tasks/main.yml

```diff
@@ -58,10 +68,10 @@
         remote_src: yes
       when: pack_location == "remote"
 
-#    - name: Copy JDBC Driver
-#      copy:
-#       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
-#       dest: "{{ carbon_home }}/repository/components/lib"
+    - name: Copy JDBC Driver to Analytics folder
+      copy:
+       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
+       dest: "{{ carbon_home }}/wso2/analytics/lib"
```

Comment:
Add a condition to check if a JDBC driver has been defined.

Refer https://github.com/wso2/ansible-apim/blob/master/roles/apim/tasks/main.yml#L95

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243